addons/mail/static/src/models/composer/composer.js

```diff
@@ -613,6 +625,33 @@ function factory(dependencies) {
             compute: '_computeMentionedPartners',
             dependencies: ['textInputContent'],
         }),
+        /**
+         * List of partners that are recipients of message being composed with
+         * this composer.
+         * This is the merge between mentionned partners and suggested
+         * recipients. It hold the people who will recive a notification.
```

Comment:
holds

Reply:
Note that this last phrase is not completely true, because followers are not included but they will receive it too. Maybe just remove it.